test/manager_test.exs

```diff
@@ -91,6 +90,20 @@ defmodule ManagerTest do
     assert %{} == Crawly.Engine.running_spiders()
   end
 
+  @tag timeout: 61_000
+  test "spider does not close after 1 minute when closespider timeout is disabled" do
+    Application.put_env(:crawly, :closespider_timeout, :disabled)
```

Comment:
@maiphuong-van What happens if you also use:
Application.put_env(:crawly, :manager_operations_timeout, 1_000)?

Reply:
I can try that 👍

Reply:
It works! I will push the update :)